core/src/main/java/io/nixer/nixerplugin/core/stigma/evaluate/StigmaValidator.java

```diff
@@ -0,0 +1,48 @@
+package io.nixer.nixerplugin.core.stigma.evaluate;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.function.Supplier;
+import javax.annotation.Nullable;
+
+import io.nixer.nixerplugin.core.stigma.domain.StigmaStatus;
+import io.nixer.nixerplugin.core.stigma.storage.StigmaData;
+
+/**
+ * Created on 21/01/2020.
+ *
+ * @author Grzegorz Cwiak (gcwiak)
+ */
+public class StigmaValidator {
+
+    private final Supplier<Instant> nowSource;
+
+    private final Duration stigmaLifetime;
+
+    public StigmaValidator(final Supplier<Instant> nowSource, final Duration stigmaLifetime) {
+        this.nowSource = nowSource;
+        this.stigmaLifetime = stigmaLifetime;
+    }
+
+    public boolean isValid(@Nullable final StigmaData stigmaData) {
```

Comment:
name change to isActive?

Reply:
It would be misleading as indicating we check the status is active, while the stigma might be invalid due to different reasons, like expired, revoked, etc.